### ISSUE TITLE
Add a x-default-value property

### ISSUE THREAD

**Issue by lowlines** [Bungie-net/api#11]
This is a suggestion to add some kind of default/sample value property. When I iterate through a data structure to build a sample json I by default set strings to "" and numbers to 0 but there are instances where having like an example value would be useful such as with definition hashes, image urls and date time strings.

**Comment by vthornheart-bng** [comment]
Yeah, I totally dig that - there's actually an "example" property built into the OpenAPI spec that I was going to use for that, but I didn't have time to actually go through and fill those out (because theoretically Example would be for complex types as well as simple ones, so there'd be a lot to fill out).

I definitely intend to do that if and when I find time, as it'd most certainly be useful!  When I do, I'll use the "example" property for it!
